aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,27 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## February 14 - March 4, 2022
+
+### Communications
+- **Office Hours** - We have combined our two office hours into one which will be held every Thursday, 10:00 PM IST / 8:00 AM PST. This office hours is open to all. Contact Kamlesh Thakur, if you would like a calendar invite.
+
+### Roadmap Updates 
+- **Page Layouts**
+  - Added support for `xsmall` breakpoint. This is a **breaking change** in the theme and requires instances where users are using small as a breakpoint to shift to mobile designs need to change their check to `xsmall`.
+  - Completed dashboard template for a [3 column dashboard layout](https://design-system.hpe.design/templates/dashboards#three-column-dashboard).
+- **Page Header** - Completed research and gathering use cases for PageHeader; gave a share out on PageHeader to the team and commenced ideating designs for PageHeader based off of use cases and feedback gathered.
+- **Banner Notification** - Completed ideation, gave a share out on Banner Notification to the Design System team. Ideation on Banner Notification on the Design System site to come.
```

Comment:
I think we want to focus on what a design system user will want to know about these topics. Wording suggestion:

"Completed ideation on Banner Notifications. Guidance, theme updates, and implementation examples for Banner Notifications will be coming soon."